google/cloud/aiplatform/training_jobs.py

```diff
@@ -908,9 +908,9 @@ def make_package(self, package_directory: str) -> str:
         shutil.copy(self.script_path, script_out_path)
 
         # Run setup.py to create the source distribution.
-        setup_cmd = self._SETUP_PY_SOURCE_DISTRIBUTION_CMD.format(
-            python_executable=_get_python_executable()
-        ).split()
+        setup_cmd = [
```

Comment:
Let me know if it's more Pythonic to use append, extend or list comprehension instead.

Reply:
Looks good to me.